server_test.go

```diff
@@ -356,19 +361,6 @@ func TestServer_DropDatabase(t *testing.T) {
 	}
 }
 
-// Ensure the server returns an error when dropping a database that doesn't exist.
-func TestServer_DropDatabase_ErrDatabaseNotFound(t *testing.T) {
```

Comment:
They were broken by my changes so I migrated them into `server_integration_test.go` instead of fixing them here.